webui/react/src/components/FilterForm/TableFilter.tsx

```diff
@@ -33,15 +33,15 @@ const TableFilter = ({
     (newOpen: boolean) => {
       if (newOpen) {
         Loadable.match(formset, {
+          _: () => {
```

Comment:
Note that technically since I haven't introduced any usages of `Failed` (outside CodeEditor) all of these code modifications could be correctly done like this. I tried to identify places where we would eventually want to handle errors vs the ones where we won't and write the code accordingly.